docs/cost-estimate/README.md

```diff
@@ -0,0 +1,200 @@
+|                  | Storage             | $0.00       | $10.69      | $83.12        | $173.66       | $357.74       | $541.82       |
+|                  | Egress              | $0.00       | $104.40     | $518.50       | $1,036.17     | $2,070.60     | $3,105.03     |
+|                  | IOPS                | $12.00      | $12.00      | $18.00        | $30.00        | $60.00        | $90.00        |
+|                  | **Total (Azure)**   | **$93.26**  | **$227.85** | **$781.99**   | **$1,441.20** | **$2,813.08** | **$4,061.59** |
+| **DigitalOcean** | Compute (vCPU)      | $42.00      | $42.00      | $84.00        | $84.00        | $168.00       | $168.00       |
+|                  | Compute (RAM)       | $16.28      | $32.56      | $32.56        | $65.12        | $65.12        | $65.12        |
+|                  | Storage             | $0.00       | $14.25      | $110.83       | $231.55       | $472.99       | $714.43       |
+|                  | Egress              | $0.00       | $11.80      | $59.50        | $119.10       | $238.00       | $356.90       |
+|                  | IOPS                | $8.00       | $8.00       | $12.00        | $25.00        | $45.00        | $65.00        |
+|                  | **Total (DO)**      | **$66.28**  | **$108.61** | **$298.89**   | **$524.77**   | **$989.11**   | **$1,369.45** |
+| **Linode**       | Compute (vCPU)      | $36.00      | $36.00      | $60.00        | $60.00        | $120.00       | $120.00       |
+|                  | Compute (RAM)       | $21.75      | $43.51      | $43.51        | $87.02        | $87.02        | $87.02        |
+|                  | Storage             | $0.00       | $0.00       | $18.43        | $139.15       | $380.59       | $622.03       |
+|                  | Egress              | $0.00       | $5.90       | $29.75        | $59.55        | $119.00       | $178.45       |
+|                  | IOPS                | $7.00       | $7.00       | $10.00        | $20.00        | $40.00        | $60.00        |
+|                  | **Total (Linode)**  | **$64.75**  | **$92.41**  | **$161.69**   | **$365.72**   | **$746.61**   | **$1,067.50** |
+| **Hetzner**      | Compute (vCPU)      | $5.40       | $5.40       | $17.80        | $17.80        | $32.90        | $32.90        |
+|                  | Compute (RAM)       | $4.98       | $9.96       | $9.96         | $19.93        | $19.93        | $19.93        |
+|                  | Storage             | $0.60       | $11.16      | $55.58        | $111.11       | $222.17       | $333.24       |
+|                  | Egress              | $0.00       | $0.00       | $6.43         | $12.87        | $25.70        | $38.54        |
+|                  | IOPS                | $5.00       | $5.00       | $8.00         | $15.00        | $30.00        | $45.00        |
+|                  | **Total (Hetzner)** | **$15.98**  | **$31.52**  | **$97.77**    | **$176.71**   | **$330.70**   | **$469.61**   |
```

Comment:
These results are consistent with the online Leios calculator if one approximately aligns the assumptions.